core/sql/parser/sqlparser.y

```diff
@@ -14684,6 +14759,11 @@ optional_limit_spec : TOK_LIMIT NUMERIC_LITERAL_EXACT_NO_SCALE
 
 dml_statement : dml_query { $$ = $1; }
 
+               | with_clause_list dml_query
```

Comment:
That is the number of conflicts encountered by the bison parser when it processes file core/sql/parser/sqlparser.y. Here is how I get this information:

```
touch core/sql/parser/sqlparser.y
cd core/sql/nskgmake
gmake -j 4 -ks linuxdebug
```

Here is the current output:

```
../parser/sqlparser.y: warning: 73 shift/reduce conflicts [-Wconflicts-sr]
../parser/sqlparser.y: warning: 12 reduce/reduce conflicts [-Wconflicts-rr]
```

If you get a different (higher) number, then we would have to investigate whether there are any potential problems with the new syntax.